src/components/molecules/search-results-pager/search-results-pager.jsx

```diff
@@ -0,0 +1,120 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import { i18n } from '../../../utils';
+
+import './search-results-pager.scss';
+
+const SearchResultsPager = ({
+	current,
+	total,
+	testid = 'tid-results-pager',
+	keyword,
+	resultsPerPage,
+	isTop = true,
+	language = 'en',
+}) => {
+	// Pagination Button
+	const PgButton = (i) => {
+		let bstate = `pager__button `;
+		if (i === current) bstate += 'active ';
+		if (i === total) bstate += 'total_pages';
+		
+		return (
+			<li className="pager__list-item" key={`pager__button-${i}`}>
+				<a
+					href={`/?swkeyword=${keyword}&from=${i}&pageunit=${resultsPerPage}`}
```

Comment:
I thought we were sticking to prod:
https://www.cancer.gov/search/results?swKeyword=nih&page=2&pageunit=10&Offset=10

with the query string pattern for pagination as such: `page=2&pageunit=10&Offset=10`
page: Actual page number
pageunit: Number of items per page (I presume)
Offset: Where the current page record starts

Reply:
Yeah the bitwise / floor hack - mainly for performance but also fixes any NAN problems.